index.html

```diff
@@ -293,29 +293,56 @@ <h3>Expected Behavior</h3>
-      For a face to face event you may need to contact venue staff for assistance contacting them.</p>
-    <p>You may also contact a W3C ombud who will assist you: <a href="mailto:ombuds@w3.org">ombuds@w3.org</a>.</p>
+    <p>If you are concerned about your immediate safety, contact 
+      <a href="https://en.wikipedia.org/wiki/List_of_emergency_telephone_numbers">local law enforcement</a>.
+      For a face to face event Chairs and venue staff should provide any necessary assistance contacting them.
+    </p>
+    
+    <p>You may also contact a W3C ombud who will assist you. Contact addresses are included in the
+      <a href="https://www.w3.org/Consortium/pwe/#ombuds">list of current W3c ombuds</a> or you can mail 
+      all ombuds collectively at <a href="mailto:ombuds@w3.org">ombuds@w3.org</a>.
+    </p>
 
     <p>If you have an issue with someone’s behaviour along the lines of this Code then please raise it, there
       are a few potential people you could raise it to depending on your situation and your safety.</p>
 
-    <p>You are welcome to raise issues directly with the <a href="mailto:ombuds@w3.org">Ombuds</a> who
+    <p>You are welcome to raise issues directly with the <a href="https://www.w3.org/Consortium/pwe/#ombuds">Ombuds</a> who
       will assist you. All complaints will be taken seriously and will receive a response.</p>
 
-    <p>If it is applicable, You may instead choose to raise issues with a chair of the group. A group chair will have
-      more context so may be better able to assist you. If they do not feel able to assist you, they should help you raise
+    <p>You may instead choose to raise issues with a chair of the group. A chair generally knows the context of the group
+      so may be better able to assist you. If they do not feel able to assist you, they should help you raise
       the issue with an Ombuds person. You may also ask for them to help you raise it to an Ombuds person immediately
       if you do not feel comfortable contacting the Ombuds directly. 
     </p>
 
-    <p>If you are responsible for a community within the W3C such as a chair of a working group and you witness harassment or
-      any other behaviour which goes against this code you are encouraged to raise issues directly with
-      an <a href="mailto:ombuds@w3.org">Ombuds</a> person who can assist you.
+    <p>If you are in a <a>leadership position</a> and you witness harassment or other 
+      <a href="#unacceptablebehavior">unacceptable behaviour</a> you should raise issues directly with
+      an <a href="https://www.w3.org/Consortium/pwe/#ombuds">Ombuds</a> person who can assist you.
     </p>
-
+    
+    <p>Any person in a <a>leadership position</a> should provide immediate assistance to any participant
+      who wants to contact an ombuds. Failure to do so may increase the harmful consequences of the action being reported.
```

Comment:
s/ombuds/[whatever generic term we use for a single Ombudsperson]

Reply:
ombuds is both a singular and plural noun. (it's short for ombudsman/ombudsmen)

Reply:
If we want to use it like that then we need to define it, because "ombud" is not an entry at all in the Oxford English Dictionary, so I would assume it is not universally understood.